CI.Jenkinsfile

```diff
@@ -0,0 +1,52 @@
+
+
+def versions = [3.11, 3.10, 3.9, 3.8, 3.7]
```

Comment:
Can you update the `classifiers` in `setup.py` to match this list.

Reply:
Updated!